modules/fedimint-mint-server/src/lib.rs

```diff
@@ -375,11 +427,11 @@ impl ServerModule for Mint {
 
         dbtx.insert_new_entry(&MintAuditItemKey::Issuance(out_point), &output.amount)
             .await;
-
-        Ok(TransactionItemAmount {
-            amount: output.amount,
-            fee: self.cfg.consensus.fee_consensus.note_issuance_abs,
-        })
+        let amount = output.amount;
+        let fee = self.cfg.consensus.fee_consensus.note_issuance_abs;
+        MINT_ISSUED_ECASH_SATS.observe(amount.sats_f64());
+        MINT_ISSUED_ECASH_FEES_SATS.observe(fee.sats_f64());
+        Ok(TransactionItemAmount { amount, fee })
```

Comment:
I'm not super happy with having this code in here between the consensus logic. Maybe it could be a separate function `fn observe_mint_input(TransactionItemAmount)`.